src/Translatable/Translatable.php

```diff
@@ -464,10 +457,50 @@ public function replicateWithTranslations(array $except = null)
      */
     protected function isTranslationDirty(Model $translation)
     {
-        $dirtyAttributes = $translation->getDirty();
-        unset($dirtyAttributes[$this->getLocaleKey()]);
+        $dirty = $translation->getDirty();
+
+        unset($dirty[$this->getLocaleKey()]);
+
+        if ($notEmpty = ! empty($dirty)) {
+            $original = [];
+
+            foreach ($dirty as $key => $value) {
+                $original[$key] = $translation->getOriginal($key);
+            }
+
+            config([
+                static::class.'.'.$this->getKey().'.'.$translation->{$this->getLocaleKey()} => [
+                    'dirty'    => $dirty,
+                    'original' => $original,
+                ],
+            ]);
+        }
+
+        return $notEmpty;
+    }
+
+    /**
+     * Get the attributes that have been changed since last sync.
+     *
+     * @return array
+     */
+    public function getDirtyWithTranslations()
+    {
+        if ($translations = config(static::class.'.'.$this->getKey())) {
```

Comment:
I haven't tested it but this method is public and relies on the config setting in a protected one but doesn't call this method itself.

In my opinion it should be the other way. That the is method is based on the get one.